tor_core/initialize.py

```diff
@@ -41,7 +42,8 @@ def configure_redis():
     :return: object: the active Redis object.
     """
     try:
-        redis_server = redis.StrictRedis(host='localhost', port=6379, db=0)
+        url = os.getenv('REDIS_CONNECTION_URL', 'redis://localhost:6379')
```

Comment:
If we're putting all the redis logic into config, do we still need it here?

Reply:
Backwards compatibility. It's a method that was publicly available before (as noted by the name of the method not starting with a `_`) so I wanted to keep this as a compatibility layer.